treeherder/seta/management/commands/reset_seta.py

```diff
@@ -1,13 +1,18 @@
 from django.core.management.base import BaseCommand
```

Comment:
With this version, `time ./manage.py load_initial_data` is fast (~2s), though doesn't it mean that after every production deploy, SETA will be reset to default values until the next daily seta run completes? Is this desired?